### DISCUSSION TITLE
Concurrent connection level

### DISCUSSION THREAD

**Discussion by wodomierz** [emqx/emqx#6607] [Q&A] [answered]
Hi!
In your business doc about differences between enterprise and broker version (https://docs.emqx.io/en/broker/v4.3/faq/enterprise.html) there is written:
```
What's the major difference between EMQ X enterprise and broker?
...
* Concurrent connection level: the stable concurrent connection level for open source version is 100k, while enterprise version is 1M.
```

My question is - how is it possible, since enterprise version uses open-source core and on your github page you are mentioning, that the emqx can hold 10M+ connections: "The 3.0 release of the EMQ X broker can scale to 10+ million concurrent MQTT connections on one cluster."

**Comment by zmstone** [comment, marked as answer]
thank you for reporting it @wodomierz 
this one outdated and should be removed.

fixed here: https://github.com/emqx/emqx-docs/pull/973
